electron/menu.ts

```diff
@@ -0,0 +1,244 @@
+        },
+      },
+      {
+        label: 'Replace',
+        click: () => {
+          store.dispatch(setMonacoEditor({...uiState.monacoEditor, replace: true}));
+        },
+      },
+      {type: 'separator'},
+      {
+        label: 'Apply',
+        accelerator: hotkeys.APPLY_SELECTION,
+        click: () => {
+          store.dispatch(setMonacoEditor({...uiState.monacoEditor, apply: true}));
+        },
+      },
+      {
+        label: 'Diff',
+        accelerator: hotkeys.DIFF_RESOURCE,
+        click: async () => {
+          const {performResourceDiff} = await import('@redux/thunks/diffResource');
+          store.dispatch(performResourceDiff(<string>mainState.selectedResourceId));
+        },
+      },
+    ],
+  };
+};
+
+const viewMenu = (win: BrowserWindow, store: any): MenuItemConstructorOptions => {
+  let mainState: AppState = store.getState().main;
+  const isPreviousResourceEnabled =
+    mainState.selectionHistory.length > 1 &&
+    (mainState.currentSelectionHistoryIndex === undefined ||
+      (mainState.currentSelectionHistoryIndex && mainState.currentSelectionHistoryIndex > 0));
+  const isNextResourceEnabled =
+    mainState.selectionHistory.length > 1 &&
+    mainState.currentSelectionHistoryIndex !== undefined &&
+    mainState.currentSelectionHistoryIndex < mainState.selectionHistory.length - 1;
+
+  return {
+    label: 'View',
+    submenu: [
+      {
+        label: 'Previous Resource',
+        accelerator: hotkeys.SELECT_FROM_HISTORY_BACK,
+        enabled: Boolean(isPreviousResourceEnabled),
+        click: async () => {
+          const {selectFromHistory} = await import('@redux/thunks/selectionHistory'); // Temporary fix until refactor
+          store.dispatch(selectFromHistory({direction: 'left'}));
+        },
+      },
+      {
+        label: 'Next Resource',
+        accelerator: hotkeys.SELECT_FROM_HISTORY_FORWARD,
+        enabled: Boolean(isNextResourceEnabled),
+        click: async () => {
+          const {selectFromHistory} = await import('@redux/thunks/selectionHistory'); // Temporary fix until refactor
+          store.dispatch(selectFromHistory({direction: 'right'}));
+        },
+      },
+      {type: 'separator'},
+      {
+        label: 'Toggle Left Menu',
+        click: () => {
+          store.dispatch(toggleLeftMenu());
+        },
+      },
+      {type: 'separator'},
+    ],
+  };
+};
+
+const windowMenu = (win: BrowserWindow, store: any): MenuItemConstructorOptions => {
+  return {
+    label: 'Window',
+    submenu: [
+      {role: 'minimize'},
+      {role: 'zoom'},
+      {type: 'separator'},
+      {role: 'front'},
+      {type: 'separator'},
+      {role: 'window'},
+    ],
+  };
+};
+
+const helpMenu = (win: BrowserWindow, store: any): MenuItemConstructorOptions => {
+  return {
+    label: 'Help',
+    submenu: [
+      {
+        label: 'Documentation',
+        click: async () => {
+          await shell.openExternal('https://kubeshop.github.io/monokle/');
```

Comment:
use the openDocumentation() function in shell.ts

Reply:
Done!